linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_ca/oval/shared.xml

```diff
@@ -0,0 +1,33 @@
+<def-group oval_version="5.11">
+  <definition class="compliance" id="smartcard_configure_ca" version="3">
+    <metadata>
```

Comment:
please use the [oval_metadata macro](https://complianceascode.readthedocs.io/en/latest/jinja_macros/oval.html?highlight=oval_metadata#template-oval_metadata) here

Reply:
will do